### DISCUSSION TITLE
Appropriate place to discuss regulatory enforcement

### DISCUSSION THREAD

**Discussion by mjbriggs** [CMSgov/price-transparency-guide#290] [Q&A] [answered]
Is github the appropriate place to raise concerns / ask questions about the enforcement of the price transparency regulation? Or would the [price transparency website](https://www.cms.gov/healthplan-price-transparency/contact-us) be a better location? 

Are there any other potential communication avenues that I am missing which are relevant to this question?

**Comment by shaselton-usds** [comment, marked as answer]
@mjbriggs Try to limit questions that are technical in nature to github. Otherwise, topics such as enforcement should be directed to the price transparency website like you've noted.  Thanks!
